website/content/api-docs/system/managed-keys.mdx

```diff
@@ -132,6 +132,45 @@ $ curl \
   a read-write session to the HSM. This is a boolean expressed as a string (e.g.
   `"true"`). This key is mainly to work around a limitation within AWS's CloudHSM v5
   pkcs11 implementation.
+  
+#### AWS KMS Parameters
+- `access_key` `(string: <required>)`: The AWS access key to use. This may also be provided in the
+`AWS_ACCESS_KEY_ID` environment variable or as part of the AWS profile from the AWS CLI or instance
```

Comment:
Does this still hold after we did the config-means-config change to g-k-w? Same elsewhere below.

Reply:
Correct, we ignore all environment variables now, so this and the AWS_SECRET_ACCESS_KEY below needs to be removed from this doc.

Reply:
The AWS environment variables still work. If the fields aren't passed in as parameters, the AWS code will look at the environment variables

Reply:
Ah you are correct, the aws go library itself will leverage them when we fall through without an access key along with a whole slew of other envs.

All except for AWS_KMS_ENDPOINT, that one is completely ignored as we set the "disallow_env_vars".